moped-editor/src/components/FileUpload/FileUploadDialogSingle.js

```diff
@@ -43,7 +43,14 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const FileUploadDialogSingle = (props) => {
+const FileUploadDialogSingle = ({
+  title,
+  dialogOpen,
+  handleClickCloseUploadFile,
+  handleClickSaveFile,
+  projectId,
+  fileTypesLookup,
+}) => {
```

Comment:
most of the updates in this file is going from `props` to declaring exactly which props

Reply:
i am a big fan of declaring exact props instead of using `props` so :heart: